recipe/deploy/writable.php

```diff
@@ -39,12 +39,16 @@
 // For chmod mode only (if is boolean, it has priority over `writable_recursive`)
 set('writable_chmod_recursive', true);
 
+// Allow absolute path in writable_dirs
+set('writable_allow_absolute_path', false);
```

Comment:
I think it’s otherkill. I assume setting writable outside deploy path is really corner case and should be ignored.